python_modules/dagster/dagster/_core/definitions/definitions_class.py

```diff
@@ -56,8 +56,8 @@ class Definitions:
     A Python module is loadable by Dagster tools if there is a top-level variable
     that is an instance of Definitions.
 
-    Definitions provides a few conveniences for dealing with resources that do not apply to
-    vanilla Dagster definitions:
+    Definitions provides a few conveniences for dealing with resources that do
+    not apply to old-style `@repository` declarations:
```

Comment:
"that do not apply to old-style `@repository` declarations:" could be a little jarring to a new user who has never encountered repositories. I would maybe preface this with something like "Before the introduction of the `Definitions` class, `@repository` was the API for bla blah. These are the main differences:".